README.md

```diff
@@ -2,20 +2,33 @@
 
 The explainer outlines the motivating use cases for the API, its key features, and how it can benefit web developers and users alike.
 
 ## Motivating Use Cases
 
-### Augmenting `window.print()`
+Although `window.print()` offers basic printing functionality through the browser's default print dialog, it is a relatively simple API. Developers' capabilities are restricted to selecting a file for printing and triggering the print dialog, which places the decision-making responsibility solely on the end user. While this is generally perceived as a right thing on the Web, users might benefit in a number of ways from shifting some of this control to the application developer.
 
-While `window.print()` is a basic function that triggers the browser's default print dialog, the `navigator.printing` API provides a more comprehensive set of methods and options for querying local printers, submitting print jobs with customization options, managing print jobs, and customizing print workflows. This extended functionality allows developers to have more control over the printing process and enables advanced use cases, such as printing specific pages or sections of a document, setting print options like paper size and orientation, and monitoring the status of print jobs.
+### Streamlining Workflows
+First and foremost, this shift calls for a more streamlined and intuitive printing experience. Developers can design print workflows that automate repetitive tasks, preconfigure print settings with respect to a particular printer's capabilities, and eliminate unnecessary user interactions.
 
-### Leveraging Progressive Web Apps
-The new Web API can motivate developers to switch to Progressive Web Apps (PWAs) by offering enhanced printing capabilities that were traditionally limited to native applications (in particular, support for Printer Redirection).
+### Knowing Your Printers
+To enable customization of settings, developers need to have knowledge about the capabilities of the printer. The API provides a convenient method for accessing this information, which enhances the printing experience by allowing developers to retrieve relevant details about what the printer can and cannot do. A few examples:
+* If the printer supports high-resolution printing, the application can automatically select the appropriate print resolution, resulting in sharper and more detailed output.
+* If the printer supports duplex printing, the application can prompt the user to print double-sided, saving paper and reducing printing time.
```

Comment:
Browser-mediated printing can do this today. Why does the application need to know that the output is being duplexed?